modules/3-ssdlc.livemd

```diff
@@ -9,7 +9,8 @@ Welcome to Part 3! This section is dedicated to discussing some of the more abst
 * [No Secrets In Code](#no-secrets-in-code)
 * [Making Secret Rotation Easy](#making-secret-rotation-easy)
 * [Rate Limiting](#rate-limiting)
-* [Principle of Least Privilege](#principle-of-least-privlege)
+* [Zero Trust Model](#zero-trust-model)
+* [Defense In Depth](#defense-in-depth)
```

Comment:
Great change! We should probably maintain the standard that subsections are sub-bullets in the Table of Contents and link out to their respective sections (see: the [OWASP Module](https://github.com/podium/elixir-secure-coding/blob/main/modules/2-owasp.livemd) for an example).